### ISSUE TITLE
Fatal Error

### ISSUE THREAD

**Issue by simonemontrucchio** [creativecommons/creativecommons-wordpress-plugin#46]
Plugin cannot be activated.

Parse error: syntax error, unexpected '[' in /membri/sitogruppoprova/wp-content/plugins/creativecommons-wordpress-plugin-2.1-beta/includes/class-creativecommons-image.php on line 298

**Comment by faberglas** [comment]
I have the latest version of WP on my own domain. My theme is Salient 7.0.1 from themeforest.net

I got three warnings on my Salient dashboard:
Redux Framework Warning
select2-css CDN unavailable. Some controls may not render properly. Please wait a few minutes, then try refreshing the page. Unable to load some remotely hosted scripts..
Redux Framework Warning
select2-js CDN unavailable. Some controls may not render properly. Please wait a few minutes, then try refreshing the page. Unable to load some remotely hosted scripts..
Redux Framework Warning
ace-editor-js CDN unavailable. Some controls may not render properly. Please wait a few minutes, then try refreshing the page. Unable to load some remotely hosted scripts.

I got this fatal eror message on my WP dashboard:
Plugin could not be activated because it triggered a fatal error.
Parse error: syntax error, unexpected '.', expecting ',' or ';' in /home/fabergla/public_html/markcorbettwilson.com/wp/wp-content/plugins/creativecommons-wordpress-plugin-2.1-beta/includes/class-creativecommons-button.php on line 9

**Comment by TheCrowned** [comment]
@simonemontrucchio This is likely to be due to the fact that you are running an outdated PHP version (< 5.4). The problem lies in the compact syntax for defining an array:

`foreach (['license_url', 'attribution_url', 'source_work_url', 'extra_permissions_url'] as $field) {`

which instead could be

`foreach ( array( 'license_url', 'attribution_url', 'source_work_url', 'extra_permissions_url' ) as $field) {`

As [explained here](https://stackoverflow.com/questions/17772534/php-difference-between-array-and) :)
However, I don't see any reason why we should be using a syntax that is not backwards-compatible, since there is another that is. I have just submitted PR #56 fixing the issue :)

**Comment by BjornW** [comment]
May I suggest to define the array outside of the loop? 

This is: 
1) more readable
2) better performance (although I doubt you'll notice) due to the array being defined each foreach call. 

Instead of:
`foreach ( array( 'license_url', 'attribution_url', 'source_work_url', 'extra_permissions_url' ) as $field) {`

I'd suggest:
```
$license_metadata = array( 'license_url', 'attribution_url', 'source_work_url', 'extra_permissions_url' );
foreach( $license_metadata as $field) {
```

**Comment by TheCrowned** [comment]
Totally! Just done that :)

**Comment by ahmadbilaldev** [comment]
Issue fixed in PR #56
